packages/ui/src/components/Passcode/index.tsx

```diff
@@ -187,29 +178,32 @@ const Passcode = ({
   );
 
   return (
-    <div className={classNames(styles.passcode, className)}>
-      {Array.from({ length }).map((_, index) => (
-        <input
-          ref={(element) => {
-            // eslint-disable-next-line @silverhand/fp/no-mutation
-            inputReferences.current[index] = element;
-          }}
-          // eslint-disable-next-line react/no-array-index-key
-          key={`${name}_${index}`}
-          name={`${name}_${index}`}
-          data-id={index}
-          disabled={isDisabled}
-          value={codes[index]}
-          className={hasError ? styles.error : undefined}
-          type="text"
-          inputMode="numeric"
-          maxLength={2} // Allow overwrite input
-          onPaste={onPasteHandler}
-          onInput={onInputHandler}
-          onKeyDown={onKeyDownHandler}
-          onFocus={onFocusHandler}
-        />
-      ))}
+    <div className={className}>
+      <div className={styles.passcode}>
+        {Array.from({ length }).map((_, index) => (
+          <input
+            ref={(element) => {
+              // eslint-disable-next-line @silverhand/fp/no-mutation
+              inputReferences.current[index] = element;
+            }}
+            // eslint-disable-next-line react/no-array-index-key
+            key={`${name}_${index}`}
```

Comment:
nit: `index` is enough for a `key`?

Reply:
true, just make it a little bit user-friendly.